docs/section.md

```diff
@@ -48,7 +48,8 @@ The following conventions apply to MFTF sections:
 -  `*Section.xml` is stored in the _Section_ directory of a module.
 -  The name format is `{Admin|Storefront}{SectionDescription}Section.xml`.
 -  Camel case is used for `<section>` elements.
-  They describe the function of the element rather than attempting to describe the selector used.
+They describe the function of the element rather than attempting to describe the selector used.
```

Comment:
Not sure if Line 51 applies to the bullet above or below. It needs to be indented if it belongs to the one above.